epictrack-web/src/components/workPlan/event/EventList.tsx

```diff
@@ -52,6 +52,7 @@ import {
 import { Restricted } from "components/shared/restricted";
 import { IButton } from "components/shared";
 import { showConfetti } from "styles/uiStateSlice";
+import { on } from "events";
```

Comment:
VS code auto import shenanigans :( Looks like there might be a couple others that aren't used either.